name: "Test: internal error"

on:
  workflow_dispatch:

jobs:
  test-internal-error:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout repo
        uses: actions/checkout@v2

      - name: Run in LXC
        id: run-in-lxc
        uses: wsdfhjxc/run-in-lxc-action@testing
        with:
          distr: fedora
          release: zzzzzz # internal error
          run: exit 0

      - name: Validate error's type
        if: ${{ failure() }}
        run: errorType=${{ steps.run-in-lxc.outputs.error-type }}
